ingq/book/interface/controller/book_controller.py

```
@@ -0,0 +1,22 @@
+from book.dto.schemas import CreateBookRequest, CreateBookResponse
+
+
+router = APIRouter(prefix="/v1", tags=["Book Router"])
+
+
+@router.post("/book", status_code=201)
+@inject
+def create_book(
+    request: Request,
+    create_book_request: CreateBookRequest,
+    book_service: BookService = Depends(Provide[Container.book_service]),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)